src/software/parameter/config/ai_control.yaml

```diff
@@ -41,3 +41,8 @@ AIControlConfig:
     default: "Halt"
     description: >-
         Specifies the refbox play that should be in use
+  PreviousRefboxPlay:
```

Comment:
How does the state machine handle this normally on startup? Is there any way we can just start with a sensible default? I'm concerned this parameter will never be used otherwise, and is also hard to set manually / mock.

Reply:
the default is HALT and our_restart=false, which is a reasonable. However, if the AIPlay is overridden, then the motionconstraints will be messed up, i.e. we're not supposed to kickoff when refboxsstate is HALT

Reply:
Hmmm I can't think of a better solution for now that wouldn't be way out of scope, so I'm fine with this for now.

Reply:
I'm very confused. Is a "play" here the refbox state? Why can't we just set the previous state to be equal to overridden state on startup?

Reply:
Also how is this intended to be used? i.e when do we need to change `PreviousRefboxPlay`, is it when we use the `PlayOverride`?

Reply:
Yes play is refbox gamestate. An example of this is `PREPARE_KICKOFF_US` followed by `NORMAL_START` will set `ourRestart` to `true`, while `PREPARE_KICKOFF_THEM` followed by `NORMAL_START` will set `ourRestart` to `false`, and that affects the motionconstraints. We need a sequence of 2 refboxgamestates to set the state machine to the correct state

Reply:
Oh. That is extremely confusing. Could we change `play` to `refbox_gamestate` or just `gamestate` everywhere? `play` has a very different semantic meaning in our codebase. 

I understand what you mean about the previous state now. Could we add this example to the description for this dynamic parameter to make it clear for people in the future?

Reply:
added